test/parallel/test-dgram-send-callback-buffer-empty-address.js

```diff
@@ -14,4 +14,4 @@ const onMessage = common.mustCall(function(err, bytes) {
   client.close();
 });
 
-client.send(buf, common.PORT, onMessage);
+client.bind(0, () => client.send(buf, client.address().port, onMessage));
```

Comment:
Not a bad idea. `onMessage()` is wrapped in a `mustCall()`, so in this case not strictly necessary. No strong preference.